lib/xapian_db/index_writers/sidekiq_writer.rb

```diff
@@ -17,19 +17,24 @@ class << self
         # Update an object in the index
         # @param [Object] obj An instance of a class with a blueprint configuration
         def index(obj, commit=true, changed_attrs: [])
-          Sidekiq::Client.enqueue worker_class, :index, :class => obj.class.name, :id => obj.id, :changed_attrs => changed_attrs
+          Sidekiq::Client.enqueue worker_class, 'index',
```

Comment:
Is this working with the queue configuration? Maybe we should use the `enqueue_to` method and also pass the configured queue?

Reply:
Maybe it works because of this line https://github.com/gernotkogler/xapian_db/blob/ffa72e83a226a49dd4f2284d6f3095ff723a14ff/lib/xapian_db/index_writers/sidekiq_worker.rb#L19
Please just make sure that you can configure a sidekiq_queue in the config file.

Reply:
Thanks. I have refactored the code and it should work now with a custom-configured queue.